source/_integrations/apsystems.markdown

```diff
@@ -0,0 +1,24 @@
+---
+title: APsystems
+description: Control and monitor your APsystems EZ1 microinverters locally without the cloud
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_release: 2024.6
+ha_category:
+  - Energy
+ha_domain: apsystems
+ha_platforms:
+  - sensor
+ha_integration_type: device
+---
+
+The APsystems {% term integration %} allows you to read the data from your [APsystems EZ1](https://emea.apsystems.com/diy/ez1/) microinverter.
+The following data is provided by the integration:
+
+- Lifetime production (Per input and in total)
+- Current production (Per input and in total)
+- Today's production (Per input and in total)
+
+Make sure the local API is activated and set to **continuously**. For that, connect to the inverter via Bluetooth using the app and go to **Settings** > **local mode** and set this to **continuously**.
```

Comment:
I do not have this app. But reading through this a second time I'm wondering about the casing. Do they start maybe with a capital letter,  **Local mode** and **Continuously**?

Reply:
I unfortunately cannot check that, but they probably are.

Reply:
Here's the best I can find, but this is an old screenshot form their docs
![image](https://github.com/home-assistant/home-assistant.io/assets/66328166/acfd2d23-a6f0-4012-b267-214f8d8e56cb)
 And that's how the app looks in German:
https://raw.githubusercontent.com/SonnenladenGmbH/APsystems-EZ1-API-HomeAssistant/main/assets/images/APsystems-Lokale-API-Aktivieren-Schritt4-6.png

Reply:
Thank you for checking. Looks like they use "normal" casing